src/ui/public/render_directive/render_directive.js

```diff
@@ -6,7 +6,8 @@ module.directive('renderDirective', function () {
   return {
     restrict: 'E',
     scope: {
-      'definition': '='
+      'definition': '=',
+      'scope': '=?'
```

Comment:
Yeah, I understand how that might be a bit awkward but I see it as documenting what you pass in and what you expect to be passed in.